### PR TITLE
Connect to the database and check it as part of /health

### PR DESCRIPTION
**Pull request by glenjamin**

### CODE REVIEW

File: test/mocha.opts [hunk 1]
```diff
@@ -1,4 +1,3 @@
 --require ./test/.setup.js
 --reporter spec
 --colors
---bail
```
Comment (romidane) [inline test/mocha.opts]: Why did you remove bail?

Comment (glenjamin) [inline test/mocha.opts]: Whoops, didn't mean to include this in the commit, will rebase it out

File: test/unit/server/routes/health.spec.js [hunk 2]
```diff
@@ -22,23 +28,33 @@ describe('GET /health', () => {
       }),
   );
 
-  context('when the build-info.json file is present', () => {
-    const projectRoot = path.resolve(__dirname, '../../../../');
-    const buildJson = path.resolve(projectRoot, 'build-info.json');
+  context('when the DB isn\'t working', () => {
```
Comment (romidane) [inline test/unit/server/routes/health.spec.js]: Noticed that you are not restoring stubs after the test suite. Is there any reason for that?.. Also usually I avoid stubs as it hides errors how come you went with this approach?

Comment (glenjamin) [inline test/unit/server/routes/health.spec.js]: As the stubs are created for the test and injected, there's nothing to clean up.

What would you use instead of stubs? Mocks? I generally prefer to use spy-type assertions at the end of the tests, and/or assert on behaviour exhibited by the stubs. The idea being that the test still mostly reads top-to-bottom and isn't overly sensitive to the exact order/arguments being called.

Comment (romidane) [inline test/unit/server/routes/health.spec.js]: for this specific case the behavior you are testing is a file being present in the file system and being displayed as part of the response. I think an integration test would be more appropriate as it cover the real world usage. But I that there is more involved to produce the this JSON so perhaps I'm over thinking this

Comment (glenjamin) [inline test/unit/server/routes/health.spec.js]: Oh, you mean not stubbing at all and just using the real thing? I do generally lean towards that but interacting with the filesystem directly felt a bit ugly for this sort of test.

There's still the REST test in the e2e pack for the health endpoint, I'll expand that to assert on some of the other bits
